src/cli/timing_tests.cpp

```diff
@@ -57,9 +57,9 @@ class Timing_Test
       virtual ~Timing_Test() {}
 
       std::vector<std::vector<ticks>>
-      execute_evaluation(const std::vector<std::string>& inputs,
-                         size_t warmup_runs,
-                         size_t measurement_runs);
+                                   execute_evaluation(const std::vector<std::string>& inputs,
```

Comment:
Can this gigantic 30 space indent for manual line breaks be turned off or be reduced to a single tab width (3 spaces)?